### PR TITLE
Build Static Assets in CI

### PR DESCRIPTION
**Pull request by DemonusPC**
Resolves #NUMBER

**Overall change:**
We are having a `single point of truth` issues throughout the CD pipeline due to recent changes. It was agreed that all build steps should be made in the CI pipeline.

**Code changes:**

- Moved the `Build static assets` step from CD into CI

---

- [ ] I have assigned myself to this PR and the corresponding issues
- [ ] Tests added for new features
- [ ] Test engineer approval

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -66,6 +67,28 @@ def notifySlack(messageParameters) {
   )
 }
 
+def envStringGen(env){
+  if (env == "live"){
+    return "live";
+  }else if (env == "test"){
```
Comment (chris-hinds) [inline Jenkinsfile]: the elseIf shouldn't be required here as this will be caught via the default return

File: Jenkinsfile [hunk 2]
```diff
@@ -66,6 +67,28 @@ def notifySlack(messageParameters) {
   )
 }
 
+def envStringGen(env){
```
Comment (chris-hinds) [inline Jenkinsfile]: is this method actually required?

Comment (DemonusPC) [reply Jenkinsfile]: Removing it now

**Review by chris-hinds — APPROVED** [review]

**Review by mbzain76 — APPROVED** [review]
Ok

**Pull request merged**